app/shared/reservation-controls/ReservationControls.js

```diff
@@ -63,6 +64,18 @@ class ReservationControls extends Component {
     };
   }
 
+  get isStaff() {
+    const { resource } = this.props;
+
+    if (!resource) {
+      return false;
+    }
+
+    const isUnitStaff = getIsUnitStaff(getUnitRoleFromResource(resource));
+
+    return isUnitStaff;
```

Comment:
```suggestion
   return getIsUnitStaff(getUnitRoleFromResource(resource));
```
No need to store value to variable.